trpcultivate_phenotypes/src/Plugin/Validators/EmptyCell.php

```diff
@@ -23,24 +18,17 @@
  */
 class EmptyCell extends TripalCultivatePhenotypesValidatorBase implements ContainerFactoryPluginInterface {
 
-  /**
+  /*
```

Comment:
They didn't change this right? It should still be 2 *'s?
```suggestion
  /**
```

Reply:
So it looks like the extension is complaining about the use of double stars here so it must have been changed automatically 🙈 

The error I'm getting is:
> Inline doc block comments are not allowed; use "/* Comment */" or "// Comment" instead

I think this has to do with documenting above `use` statements and not a property or method, for which the use of `/**` IS enforced. I did a search for`/*` and it only occurred in docblocks above these use statements in each validator (within the files touched by this PR - **6 occurrences**).

Reply:
I did go in and clean up these docblocks for all the validators (6 in total use traits) in commit [87c758b](https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/119/commits/87c758be88843cb3b6d01dcbbad1c2ffbd2e7030)

They now meet the standards for typical docblocks, particularly the first line being a short description on its own. What do you think? I think I'm okay with the the use of a single star in these niche cases but if you would prefer to make an exception to allow us to use it here too, I'm also good with that.

Reply:
I feel strongly about the /** */ docblock notation for these use statements. When you fixed it for the validators, it didn't show as an error/problem with the two stars? I'm not sure how hard it would be to add in an exception for the autoformatting 🤔  and if it doesn't show up as a problem then maybe it is the intellephense making the fix, not phpcs 😬

Reply:
After discussion virtually this became more clear to me. PR https://github.com/TripalCultivate/TripalCultivate/pull/32 in the base module makes the change to the coding standards so that we can keep our ** here :-)

Reply:
Re-added ** in commit [9c6f567](https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/119/commits/9c6f567d684d387d84a9610e068af53466e3d95f)!